### PR TITLE
dockerfile and stress.sh for Memory-stress testing.

### PR DESCRIPTION
**Pull request by weezer**

### CODE REVIEW

**Comment by jacobwagner** [comment]
Please squash your commits into a single commit

File: memory-stress/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,6 @@
+FROM nginx
+RUN apt-get update && apt-get install -y --force-yes stress
+EXPOSE 80
+COPY ./stress.sh .
+RUN chmod +x ./stress.sh
+ENTRYPOINT ["./stress.sh"]
```
Comment (jacobwagner) [inline memory-stress/Dockerfile]: newline please. should have this set in your editor

File: memory-stress/stress.sh [hunk 2]
```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+
+echo "Hello Kubernetes" > /usr/share/nginx/html/index.html
+nginx &
+sleep `tr -cd 0-9 </dev/urandom | head -c 2`
+stress --vm 4 --timeout 100 --vm-bytes 6000M
```
Comment (jacobwagner) [inline memory-stress/stress.sh]: newline please, you should set this in your editor

**Review by jacobwagner — CHANGES_REQUESTED** [review]
add newlines please

**Review by EllieSky — APPROVED** [review]
:shipit:

**Pull request merged**